src/plus/integrations/integrationService.ts

```diff
@@ -857,12 +932,33 @@ export class IntegrationService implements Disposable {
 		return this._integrations.get(this.getCacheKey(id, domain));
 	}
 
+	private findCachedById(id: SupportedSelfHostedIntegrationIds): Integration | undefined {
+		const key = this.getCacheKey(id, '');
+		for (const [k, integration] of this._integrations) {
+			if (k.startsWith(key)) {
+				return integration;
+			}
+		}
+		return undefined;
+	}
+
```

Comment:
@axosoft-ramint doesn't this still need to be fixed up to look in the configured set?

Reply:
@eamodio I added the configured set lookup in the case statement for `CloudGitHubEnterprise` in the `get` function of this file, rather than here. But we could just as easily move that lookup into this function.